worlds/stardew_valley/test/checks/option_checks.py

```diff
@@ -80,11 +63,10 @@ def assert_has_deluxe_scarecrow_recipe(tester: SVTestBase, multiworld: MultiWorl
 
 
 def assert_festivals_give_access_to_deluxe_scarecrow(tester: SVTestBase, multiworld: MultiWorld):
-    has_festivals = is_not_setting(multiworld, options.FestivalLocations.internal_name, options.FestivalLocations.option_disabled)
+    sd_options = get_stardew_options(multiworld)
+    has_festivals = sd_options.festival_locations.value == options.FestivalLocations.option_disabled

```

Comment:
This is an error. The condition was for **not** being disabled, as in, having any festival setting. But the new condition is for being disabled instead.

It should be "!=" instead